docs/tutorial/using-selenium-and-webdriver.md

```diff
@@ -86,12 +86,12 @@ const driver = new webdriver.Builder()
   // The "9515" is the port opened by chrome driver.
   .usingServer('http://localhost:9515')
   .withCapabilities({
-    chromeOptions: {
+    'goog:chromeOptions': {
```

Comment:
Same question here as for 13: was `goog:chromeOptions` the right property name all along, or did it change? And if it changed, is `goog.chromeOptions` the correct key for Electron 12?

Reply:
I just checked the history for this doc and https://github.com/electron/electron/pull/28231 specified that this was required for the latest versions of `selenium-webdriver`, and that the new property name still worked with older versions.